src/main/java/com/github/lunatrius/ingameinfo/tag/TagTime.java

```diff
@@ -82,6 +99,7 @@ public String getValue() {
     }
 
     public static void register() {
+        TagRegistry.INSTANCE.register(new MinecraftTotal().setName("total"));
```

Comment:
Please use a less ambigous name

Reply:
Oh sorry, I'll change that and squash it. Thanks for the quick response